gdal/frmts/wcs/frmt_wcs.html

```diff
@@ -47,9 +47,9 @@
 <li> <b>BlockYSize</b>: The block height to use for block cached remote access.
 <li> <b>NoDataValue</b>: The nodata value to use for all the bands (blank for none).  Normally defaulted from CoverageOffering info.
 <li> <b>Timeout</b>: The timeout to use for remote service requests.  If not provided, the libcurl default is used.
-<li> <b>UserPwd</b>: May be supplied with <i>userid:password</i> to pass a
+<li> <b>USERPWD</b>: May be supplied with <i>userid:password</i> to pass a
```

Comment:
To me it looks that the use of UserPwd in the docs was a typo since it is upper case in the code and it is a HTTP parameter that is upper case everywhere else and the other HTTP parameter below it is upper case.

Reply:
I indeed see CPLGetXMLValue( psService, "USERPWD", NULL ) in the code, but looking at CPLGetXMLValue() / CPLGetXMLNode() it uses EQUAL() which is case insensitive. So both will work. We just need to be consistent in the way we at least document this (and in the code as well preferably)